lib/global-styles.php

```diff
@@ -488,18 +542,18 @@ function gutenberg_experimental_global_styles_enqueue_assets() {
  * @return array New block editor settings
  */
 function gutenberg_experimental_global_styles_settings( $settings ) {
-	if ( ! gutenberg_experimental_global_styles_has_theme_json_support() ) {
```

Comment:
Is this necessary here? I'd rather keep it at the moment.

Reply:
We need it for the link color, in order to provide a default link color value.

Reply:
We will need default values for the new UI's we provide and these values need to be present independently if the theme supports global styles or not.